composer.json

```
@@ -28,7 +28,7 @@
         }
     },
     "require": {
-        "php": "~8.0.0 || ~8.1.0 || ~8.2.0 || ~8.3.0",
+        "php": "~8.0.0 || ~8.1.0 || ~8.2.0 || ~8.3.0 || ~8.4.0",
```

[Version 8.0 is no longer supported:](https://www.php.net/supported-versions)

```suggestion
        "php": "~8.1.0 || ~8.2.0 || ~8.3.0 || ~8.4.0",
```

---

Done